plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcConnectorTest.java

```diff
@@ -22,4 +24,26 @@ public TestIcebergOrcConnectorTest()
     {
         super(ORC);
     }
+
+    @Override
+    protected boolean supportsIcebergFileStatistics(String typeName)
+    {
+        return !(typeName.equalsIgnoreCase("boolean") ||
+                typeName.equalsIgnoreCase("varbinary") ||
+                typeName.contains("timestamp"));
+    }
+
+    @Override
+    protected boolean supportsRowGroupStatistics(String typeName)
+    {
+        return !typeName.equalsIgnoreCase("varbinary");
+    }
+
+    @Override
+    protected Session withSmallRowGroups()
```

Comment:
nit: it's slightly more readable (and flexible) to pass initial `Session` as an arg